contexts/generic.jsonld

```diff
@@ -41,7 +41,7 @@
         },
         "responseOptions": {
             "@id": "repronim:valueconstraints",
-            "@type": "@id"
+            "@type": "@vocab"
```

Comment:
@sanuann for me to reference "dontKnow" from the context, i needed to change responseOptions's "@type" to "@vocab". i hope this is fine